source/_integrations/renault.markdown

```diff
@@ -37,6 +37,24 @@ This integration provides the following platforms:
 
 All vehicles linked to the account should then get added as devices, with sensors added as linked entity.
 
+## Buttons
+
+### Button `button.renault_start_charge`
+
+Start charging the vehicle.
```

Comment:
maybe it is sufficient to insert a line between 31/32

Reply:
done

Reply:
not done

Reply:
got an error when trying to push. sorry